app/containers/Vaults/saga.js

```diff
@@ -94,31 +101,15 @@ const mapNewApiToOldApi = (oldVaults, newVaults) => {
 };
 
 function* fetchVaults() {
-  const endpoint =
-    process.env.API_ENV === 'development' ||
-    process.env.NODE_ENV === 'development'
-      ? `https://dev.vaults.finance/all`
-      : `https://vaults.finance/all`;
   const newEndpoint = 'https://api.yearn.finance/v1/chains/1/vaults/all';
   try {
-    const vaults = yield call(request, endpoint);
     const newVaults = yield call(request, newEndpoint);
 
-    // TODO: Remove UI hacks...
-    const masterChefAddress = '0xbD17B1ce622d73bD438b9E658acA5996dc394b0d';
```

Comment:
This is not necessary. This is only used for the special pickle vault and it's overridden in current UI anyways. Instead we display "v1" for vault type as we do today.